### PR TITLE
build: make it faster by paralleleizing

### PR DESCRIPTION
**Pull request by glimchb**
Signed-off-by: Boris Glimcher <Boris.Glimcher@emc.com>

### CODE REVIEW

**Review by mestery — APPROVED** [review]
Ohhhhh, fancy! ;)

**Comment by glimchb** [comment]
@mestery I was thinking to add another option `integration.sh` to only build... but wasn't sure... WDYT ?

**Comment by mestery** [comment]
> @mestery I was thinking to add another option `integration.sh` to only build... but wasn't sure... WDYT ?

I like this! Break out the `build-start` option to be `build` and `start` separate!

**Comment by glimchb** [comment]
> > @mestery I was thinking to add another option `integration.sh` to only build... but wasn't sure... WDYT ?
> 
> I like this! Break out the `build-start` option to be `build` and `start` separate!

ok adding now to this PR as separate commit, please re-review

File: integration/scripts/integration.sh [hunk 1]
```diff
@@ -19,11 +20,15 @@ fi
...
 usage() {
     echo ""
-    echo "Usage: integration.sh [${INT_BUILD_START} | ${INT_RUN_TESTS} | ${INT_LOGS} | ${INT_STOP}]"
+    echo "Usage: integration.sh [${INT_BUILD} | ${INT_BUILD_START} | ${INT_RUN_TESTS} | ${INT_LOGS} | ${INT_STOP}]"
     echo ""
 }
 
-build_and_start_containers() {
+build_containers() {
+    bash -c "${DC} -f docker-compose.yml -f docker-compose.xpu.yml -f docker-compose.otel.yml -f docker-compose.spdk.yml -f docker-compose.pxe.yml build --parallel"
```
Comment (seroyer) [inline integration/scripts/integration.sh:28]: I'm not going to stop this here, but I'd like to have a discussion around linting for scripts.  For example, 80 character line limits?

**Review by seroyer — APPROVED** [review]
I like it!  Do you have an estimate of the time savings?

**Comment by glimchb** [comment]
> I like it! Do you have an estimate of the time savings?

from 16m to 8m savings... on avg...

File: integration/scripts/integration.sh (hunk 1 above)
Comment (glimchb) [reply integration/scripts/integration.sh:28]: agree, we should do it

**Review by mestery — APPROVED** [review]

**Pull request merged**